scripts/modelbench/inductor_train.sh

```diff
@@ -0,0 +1,53 @@
+    for torchbench_model in ${torchbench_model_list[@]}
+    do
+        # Commands for torchbench for device=cpu, dtype=float32 for training and for performance testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/torchbench.py --performance --float32 -dcpu  --training --inductor   --no-skip --dashboard --only ${torchbench_model} --cold_start_latency --output=${LOG_DIR}/multi_threads_cf_${timestamp}_perf.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+        # Commands for torchbench for device=cpu, dtype=float32 for training and for accuracy testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/torchbench.py --accuracy --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${torchbench_model} --output=${LOG_DIR}/multi_threads_cf_${timestamp}_acc.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log             
+    done
+
+    for huggingface_model in ${huggingface_model_list[@]}
+    do
+        # Commands for huggingface for device=cpu, dtype=float32 for training and for performance testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/huggingface.py --performance --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${huggingface_model} --cold_start_latency --output=${LOG_DIR}/multi_threads_cf_${timestamp}_perf.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+        # Commands for huggingface for device=cpu, dtype=float32 for training and for accuracy testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/huggingface.py --accuracy --float32 -dcpu  --training --inductor   --no-skip --dashboard --only ${huggingface_model}  --output=${LOG_DIR}/multi_threads_cf_${timestamp}_acc.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+    done
+
+    for timm_model in ${timm_model_list[@]}
+    do
+        # Commands for timm_models for device=cpu, dtype=float32 for training and for performance testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/timm_models.py --performance --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${timm_model} --cold_start_latency --output=${LOG_DIR}/multi_threads_cf_${timestamp}_perf.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+        # Commands for timm_models for device=cpu, dtype=float32 for training and for accuracy testing
+        python -m torch.backends.xeon.run_cpu --node_id 0 benchmarks/dynamo/timm_models.py --accuracy --float32 -dcpu --training --inductor   --no-skip --dashboard --only ${timm_model} --output=${LOG_DIR}/multi_threads_cf_${timestamp}_acc.csv 2>&1 | tee -a ${LOG_DIR}/multi_threads_model_bench_log_${timestamp}.log
+    done       
+    
+elif [[ $CHANNELS == "last" ]]; then
+    # channels last
+    echo "Skip Channels last testing...."
```

Comment:
Why we need skip channels last test? For channels format can refer https://github.com/chuanqi129/inductor-tools/blob/main/scripts/modelbench/inductor_single_run.sh#L15-L18 to reduce redundant code

Reply:
Thanks, added.